sporkfish/searcher/minimax.py

```diff
@@ -228,9 +228,6 @@ def _iterative_deepening(
         for depth in range(1, self._searcher_config.max_depth + 1):
             new_board = copy.deepcopy(board)
 
-            alpha = -float("inf")
```

Comment:
Not too sure why we don't need these two lines now, but given consistency tests still pass, seems good to me.